metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -3605,6 +3616,19 @@ def get_last_modified(
             )
         return last_modified
 
+    def _get_owner_identifier(self, owner_dict: dict) -> Optional[str]:
+        """Extract owner identifier (email or username) based on configuration."""
+        if not owner_dict:
+            return None
+
+        if self.config.use_email_as_username:
+            email = owner_dict.get(c.EMAIL)
+            if email:
+                return email
+            # Fall back to username if email is not available
```

Comment:
should we have some visibility about this fallback?
a log may become too repetitive... maybe a counter?